src/test/java/Exam1Test.java

```diff
@@ -0,0 +1,66 @@
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.*;
+
+class Exam1Test {
+
+    @DisplayName("쉼표(,) 또는 콜론(:) 구분자를 기준으로 분리한 각 숫자의 합을 반환한다")
+    @ParameterizedTest
+    @CsvSource(value = {"1,2=3", "1,2,3=6","1,2:3=6"}, delimiter = '=')
+    void splitAndSumTest(String input, int expected){
+        //given
+        //String str = "1,23";
```

Comment:
given 구절을 살리고 싶으면 
```suggestion
   //given 
   String str = input;
```
다음과 같이 작성해도 좋을꺼 같아요!